fiftyone/utils/eval/detection.py

```diff
@@ -135,6 +139,9 @@ def evaluate_detections(
         progress (None): whether to render a progress bar (True/False), use the
             default value ``fiftyone.config.show_progress_bars`` (None), or a
             progress callback function to invoke instead
+        batch_size (None): the batch size at which to process samples. By
+            default, all samples are processed in a single (1) batch
```

Comment:
It's not true that the default is a batch size of 1. `iter_samples(autosave=True)` uses a batching strategy by default:
https://github.com/voxel51/fiftyone/blob/67bdf7e60a752b2686c2050d4f7ec16141fef988/fiftyone/core/view.py#L500-L516 

The default batcher sends requests every 0.2 seconds:
https://github.com/voxel51/fiftyone/blob/67bdf7e60a752b2686c2050d4f7ec16141fef988/fiftyone/core/config.py#L159-L182

Reply:
Ah, so if batch_size is not passed, we use latency strategy by default 👍